### PR TITLE
dash/add-message-on-visual-diff

### PR DESCRIPTION
**Pull request by goransle**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206286261825080

### CODE REVIEW

File: test/cypress/dashboards/visual.config.mjs [hunk 1]
```diff
@@ -24,6 +42,60 @@ export default defineConfig({
...
+                            ...commonPath
+                        )
+                    ).catch(() => []);
+
+                    const diffFiles = await walkDir(diffDir)
+                        .catch(() => []);
+
+                    if (diffFiles.length){
+
+                        console.log(diffFiles);
```
Comment (karolkolodziej) [inline test/cypress/dashboards/visual.config.mjs:66]: Do we need it?

File: test/cypress/dashboards/visual.config.mjs [hunk 2]
```diff
@@ -24,6 +42,60 @@ export default defineConfig({
...
             getCompareSnapshotsPlugin(on, config);
+
+            if(config.env.type === 'actual') {
+                on('after:run', async (results) => {
+                    // create json of files in the diff directoryA
+                    const commonPath = ['test', 'cypress', 'dashboards',  'visual'];
+
+                    const diffDir = join(config.env.SNAPSHOT_DIFF_DIRECTORY, ...commonPath);
+
+                    // TODO: try to figure out why these files are saved to this bonkers place
```
Comment (karolkolodziej) [inline test/cypress/dashboards/visual.config.mjs:53]: Is it going to be solved here or in the future?

**Review by karolkolodziej — APPROVED** [review]

**Comment by karolkolodziej** [comment]
@goransle any idea why we get the diffs in this PR?

**Comment by goransle** [comment]
> @goransle any idea why we get the diffs in this PR?

Likely due to setting the viewport sizes within the tests (and then not waiting for the dashboard to adjust). Might be better for stability to use [this approach](https://docs.cypress.io/api/commands/viewport#Set-viewport-in-the-test-configuration), but then we might miss testing dynamic responsiveness 🤔 If we want to keep that aspect we will have to find a way to wait for the dashboard to resize.

**Comment by karolkolodziej** [comment]
We are going to keep an eye on that 😉 thanks!

**Comment by goransle** [comment]
Note that the comparison is failing on this PR as the config and plugin version differs from master (aka the path to the base snapshot has changed).

**Pull request merged**